Scripts/create_script.php

```diff
@@ -25,6 +25,15 @@
 
 if ($conn->query($sql) == True) {
   print("Connection Done");
+  
+  // Only update log if query is successful
+  $sqlID = "SELECT `ID` FROM `PortfolioTracker`.`Users` WHERE `Users`.`Email`='$email';";
+  $uID = $conn->query($sqlID);
```

Comment:
$uID returns a query object, so you have to call fetch_row() to retrieve a row of the actual query result. It could look something like this:

$uID = $uID->fetch_row();
 
Then you'll have to index $uID in the sql query statement to add it to the database. It should only return one value, so it should just be $uID[0]